cmd/config.go

```diff
@@ -309,6 +301,85 @@ type CAADistributedResolverConfig struct {
 	Proxies     []string
 }
 
+type TemporalLogDescription struct {
+	URI         string
+	Key         string
+	WindowStart string
+	start       time.Time
+	WindowEnd   string
+	end         time.Time
+}
+
+type TemporalSet struct {
+	Name   string
+	Shards []TemporalLogDescription
+}
+
+// Setup initializes the TemporalSet by parsing the start and end dates
+// and verifying the intervals
```

Comment:
I think it would be clearer if you said "and verifying the begin date is before and not equal to the end date."